### DISCUSSION TITLE
Install gromacs with deepmd

### DISCUSSION THREAD

**Discussion by cui-hq** [deepmodeling/deepmd-kit#2958] [Q&A]
I want to know if the dp_gmx_patch installed with conda can be used to patch the groamcs？
My Error:  "runtime library [libgomp.so.1] in /usr/lib/gcc/x86_64-linux-gnu/8 may be hidden by files in: ~/anaconda3/envs/deepmd/lib"

**Comment by njzjz** [comment]
You may need to use the same gcc compiler (so openmp library is the same), otherwise you'll get this message.
